lib/television.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+// Define class
+const television = {
+
+  // Variables
+  powerOn: false,
+  volume: 10,
+  maxVol: 100,
+  minVol: 0,
+  maxChan: 500,
+  minChan: 0,
+  channel: 100,
+  mute: true,
+
+  // Methods
+  togglePower: function(){
+    this.powerOn = !this.powerOn;
+    console.log('Power is on: ' + this.powerOn);
```

Comment:
This is good for practice. Please don't ever push `console.log` in shipped code, unless it is a user-facing feature (users are not always developers). That includes your projects.